docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
 
```

Comment:
Do we have resources to have somewhat better grapghics? If not, can we at least use the Polkadot font? a proper background color that's Polkadot themed?

Reply:
Hi @kianenigma I am using google docs to create the graphics and it does not allow to install the Polkadot font. I will need to see what are other options with W3F. I initially produced the images with Polkadot branding colour as background but I switched to a white background using the branding colour for the font. It seemed "cleaner" to me. But I can revert it. What do you mean exactly with "better graphics"? Is the overall presentation of the diagrams not good, or just the font and the background colour?

Reply:
Let me check with the design team on this. We were already thinking about revamping the graphics on the Wiki

Reply:
I spoke to Iggy and he told me to use Inter font for the figures. Polkadot unbounded font is only used for titles. I updated all figures accordingly.